modules/standard/VisualDebug.chpl

```diff
@@ -31,6 +31,10 @@ module VisualDebug
 
   use String;
 
+  // VDebugOn -- Should we generate data
+  config param DefaultVDebugOn = true;
+  config var VDebugOn = DefaultVDebugOn;
```

Comment:
'VDebugOn' seems slightly terse. Would you mind changing it to 'VisualDebugOn' to make it more obvious what it does?